projects/Mallard/src/authentication/__tests__/helpers.spec.ts

```diff
@@ -162,4 +187,64 @@ describe('helpers', () => {
             expect(membershipStore.set).not.toBeCalled()
         })
     })
+
+    describe('canViewEdition', () => {
+        it('allows people in with guardian email addresses', () => {
+            expect(
+                canViewEdition(
+                    withCreds({
+                        email: 'alice@guardian.co.uk',
+                        digitalPack: false,
+                    }),
+                ),
+            ).toBe(true)
+
+            expect(
+                canViewEdition(
+                    withCreds({
+                        email: 'bob@theguardian.com',
+                        digitalPack: false,
+                    }),
+                ),
+            ).toBe(true)
+
+            expect(
+                canViewEdition(
+                    withCreds({
+                        email: 'charlotte@elgrauniad.biz',
+                        digitalPack: false,
+                    }),
+                ),
+            ).toBe(false)
+        })
+
+        it('allows anyone to login with a digital pack', () => {
+            expect(
+                canViewEdition(
+                    withCreds({
+                        email: 'alice@guardian.co.uk',
+                        digitalPack: true,
+                    }),
+                ),
+            ).toBe(true)
+
+            expect(
+                canViewEdition(
+                    withCreds({
+                        email: 'bob@theguardian.com',
+                        digitalPack: true,
+                    }),
+                ),
+            ).toBe(true)
+
+            expect(
+                canViewEdition(
+                    withCreds({
+                        email: 'charlotte@elgrauniad.biz',
```

Comment:
might want to use one of our non email domains here?

Reply:
Good shout, done